pkgs/by-name/di/discover-overlay/package.nix

```
@@ -0,0 +1,42 @@
+{ lib, python3, fetchFromGitHub, gtk3, gobject-introspection, wrapGAppsHook }:
+python3.pkgs.buildPythonApplication rec {
+  pname = "discover-overlay";
+  version = "0.7.0";
+
```

```suggestion
  pyproject = true;

```

This falls back to setuptools if needed.